nyc_data/ppe/models.py

```diff
@@ -161,27 +161,22 @@ class Purchase(BaseModel):
 
     raw_data = JSONField()
 
-    # property so we can use it in templates
+    @property
```

Comment:
Might be worth considering `django.utils.functional.cached_property` for something that feels expensive. This may not be the case, but worth a note.

Reply:
TIL!